src/org/opendatakit/briefcase/model/BriefcaseAnalytics.java

```diff
@@ -0,0 +1,179 @@
+package org.opendatakit.briefcase.model;
+
+import com.brsanthu.googleanalytics.EventHit;
+import com.brsanthu.googleanalytics.GoogleAnalytics;
+import com.brsanthu.googleanalytics.GoogleAnalyticsConfig;
+import org.opendatakit.briefcase.buildconfig.BuildConfig;
+
+/**
+ *  This class is used to send usage information to ODK's Google Analytics account.
+ *
+ * <P>A library built by 'brsanthu' is utilized to streamline the
+ * process of contacting Google's Measurement Protocol API.</P>
+ * <P>This class also implements the Singleton pattern.</P>
+ *
+ * @author Joao Paulo Knox
+ * @see <a href="https://github.com/brsanthu/google-analytics-java">google-analytics-java</a>
+ */
+public class BriefcaseAnalytics {
+
+  private static final String GOOGLE_TRACKING_ID = BriefcasePreferences.GOOGLE_TRACKING_ID;
+  private static final String UNIQUE_USER_ID = BriefcasePreferences.getUniqueUserID();
+  private static final String EVENT_CATEGORY = "ODK Briefcase";
+  private static final String APPLICATION_NAME = BuildConfig.NAME;
+  private static final String APPLICATION_VERSION = BuildConfig.VERSION;
+
+  private static BriefcaseAnalytics instance;
+  private static final GoogleAnalyticsConfig CONFIG = new GoogleAnalyticsConfig();
+  private static GoogleAnalytics googleAnalytics = new GoogleAnalytics(CONFIG, GOOGLE_TRACKING_ID);
+  private static boolean hasTrackedStartup;
+
+  /**
+   * Constructor
+   *
+   * <P>This constructor will instantiate 'brsanthu's {@link GoogleAnalytics} class.</P>
```

Comment:
You are mistaken, my new friend, as I demonstrate in the debugger here.

<img width="871" alt="screen shot 2017-08-15 at 12 00 45 pm" src="https://user-images.githubusercontent.com/55099/29331585-cacd1f18-81b1-11e7-9af4-094684f30fa5.png">

It is created when this line is executed from ```MainBriefcaseWindow```:
```  private static BriefcaseAnalytics briefcaseAnalytics = BriefcaseAnalytics.getInstance();```

But it’s moot now because you have removed the comment.